book/src/validator-manager-create.md

```diff
@@ -0,0 +1,175 @@
+move the `validators.json` and `deposits.json` files to an Internet-connected
+host. This would help protect the mnemonic from being exposed to the Internet.
+
+### 1. Create the Validators
+
+Run the `create` command, subsituting `<ADDRESS>` for an execution address that
+you control. This is where all the staked ETH and rewards will ultimately
+reside, so it's very important that this address is secure, acessible and
+backed-up. The `create` command:
+
+```bash
+lighthouse \
+    validator-manager \
+    create \
+    --first-index 0 \
+    --count 2 \
+    --eth1-withdrawal-address <ADDRESS> \
+    --output-path ./
+```
+
+If successful, the command output will appear like below:
+
+```bash
+Running validator manager for mainnet network
+
+Enter the mnemonic phrase:
+<REDACTED>
+Valid mnemonic provided.
+
+Starting derivation of 2 keystores. Each keystore may take several seconds.
+Completed 1/2: 0x8885c29b8f88ee9b9a37b480fd4384fed74bda33d85bc8171a904847e65688b6c9bb4362d6597fd30109fb2def6c3ae4
+Completed 2/2: 0xa262dae3dcd2b2e280af534effa16bedb27c06f2959e114d53bd2a248ca324a018dc73179899a066149471a94a1bc92f
+Keystore generation complete
+Writing "./validators.json"
+Writing "./deposits.json"
+```
+
+This command will create validators at indices `0, 1`. The exact indices created
+can be influenced with the `--first-index` and `--count` flags. Use these flags
+with caution to prevent creating the same validator twice, this may result in a
+slashing!
+
+The command will create two files:
+
+- `./deposits.json`: this file does *not* contain sensitive information and may be uploaded to the [Ethereum Staking Launchpad].
+- `./validators.json`: this file contains **sensitive unencrypted validator keys, do not share it with anyone or upload it to any website**.
+
+### 2. Import the validators
+
+The VC which will receive the validators needs to have the following flags at a minimum:
+
+- `--http`
+- `--unencrypted-http-transport`
+- `--http-address 127.0.0.1`
+- `--http-port 5062`
+- `--enable-doppelganger-protection`
+
+Therefore, the VC command might look like:
+
+```bash
+lighthouse \
+    vc \
+    --http \
+    --unencrypted-http-transport \
+    --http-address 127.0.0.1 \
+    --http-port 5062 \
+    --enable-doppelganger-protection
+```
+
+In order to import the validators, the location of the VC `api-token.txt` file
+must be known. The location of the file varies, but it is located in the
+"validator directory" of your data directory. For example:
+`~/.lighthouse/mainnet/validators/api-token.txt`. We will use `<API-TOKEN-PATH>`
+to subsitute this value.
+
+
+Once the VC is running, use the `import` command to import the validators to the VC:
+
+```bash
+lighthouse \
+    validator-manager \
+    import \
+    --validators-file validators.json \
+    --vc-token <API-TOKEN-PATH>
+```
+
+If successful, the command output will appear like below:
+
+```bash
+Running validator manager for mainnet network
```

Comment:
I think this is a good suggestion, however we have a precedence of displaying the network for all commands. This is because can be difficult to know for certain when the network is relevant. Without the ability to test to ensure commands are always network-independent we run the risks of making future changes without remembering to add the network printout.

So, I'm tempted to leave the network logging here just to be consistent. It's also safer (I think) to try and encourage users to use the network flag, even when they might not need to.

Reply:
With the `--network` flag specified as in the revised doc, yes I think it is a good idea to keep the network name